detect-nuget-inspector/detect-nuget-inspector/DependencyResolution/PackagesConfig/PackagesConfigResolver.cs

```diff
@@ -55,9 +60,14 @@ private List<NugetDependency> GetDependencies()
                 var versionRange = new NuGet.Versioning.VersionRange(version, true, version, true);
                 var framework = NuGet.Frameworks.NuGetFramework.Parse(packageRef.TargetFramework.Framework);
 
+                bool excludeDevDependency = isDevDependency && packageRef.IsDevelopmentDependency;
+
                 //TODO: Check that this works.
```

Comment:
While we're here, if we know that this works, we should probably remove this comment. :)